sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12.integration.service.ts

```diff
@@ -43,19 +47,32 @@ export class IER12IntegrationService extends SFTPIntegrationBase<void> {
       ierFileDetail.assessmentId = ierRecord.assessmentId;
       ierFileDetail.disbursementId = ierRecord.disbursementId;
       ierFileDetail.applicationNumber = ierRecord.applicationNumber;
-      ierFileDetail.institutionStudentNumber =
-        ierRecord.institutionStudentNumber;
+      ierFileDetail.institutionStudentNumber = replaceLineBreaks(
+        ierRecord.institutionStudentNumber,
+      );
       ierFileDetail.sin = ierRecord.sin;
-      ierFileDetail.studentLastName = ierRecord.studentLastName;
-      ierFileDetail.studentGivenName = ierRecord.studentGivenName;
+      ierFileDetail.studentLastName = replaceLineBreaks(
```

Comment:
From the moment that we assume that data like last name (and some others) potentially will have new lines characters I believe that we should sanitize before it is saved to DB.
For data like program description I would be ok to leave the sanitization for a later moment if the intention is to allow the user to enter the line breaks, this would be optional, but institutionStudentNumber, studentLastName, studentGivenName, addressLine1, addressLine2, city, postalCode, programName, institutionProgramCode, must be saved properly.

Reply:
I get your point and I believe that eventually all `free text not text areas` must be sanitized before going to DB.